crypto/rsa/rsa_impl.c

```diff
@@ -644,11 +635,10 @@ static int mod_exp(BIGNUM *r0, const BIGNUM *I, RSA *rsa, BN_CTX *ctx) {
     goto err;
   }
 
-  /* Turn BN_FLG_CONSTTIME flag on before division operation */
-  pr1 = &local_r1;
-  BN_with_flags(pr1, r1, BN_FLG_CONSTTIME);
+  /* Ensure BN_FLG_CONSTTIME flag is on before division operation */
+  assert(BN_get_flags(r1, BN_FLG_CONSTTIME));
```

Comment:
r1 is a local variable and BN_set_flags(r1, BN_FLG_CONSTTIME) is never set. So, won't this assertion always fail? And if so, does that mean this code isn't being run during the tests at all?